apps/03TicTacToe.js

```diff
@@ -91,3 +158,37 @@ if (typeof describe === 'function') {
   getPrompt();
 
 }
+//checks to see if there is a horizontal/vertical/diagonal win (not necessary, but to pass the ACA checks
+function horizontalWin() {
+
+	for (var i=0; i<3; i++){
+	      if (board[i].join("")==="XXX" || board[i].join("")==="OOO"){
+		return true;
+      	}
+      }
+      return false;
+    }
+
+function verticalWin() {
+	var varCheck="";
+	for (var i=0; i<3; i++){
+		for (var j=0;j<3; j++){
+		varCheck += board[j][i];
+		if (varCheck ==="XXX" || varCheck==="OOO"){
+		return true;
+      		}
+	      }
+		varCheck="";
+      }
+      return false;
+    
+}
+
+function diagonalWin() {
+var checkOne = board[0][0]+board[1][1]+board[2][2]+"";
+var checkTwo = board[0][2]+board[1][1]+board[2][0]+"";
+if(checkOne === "XXX" || checkOne === "OOO" || checkTwo === "OOO" || checkTwo === "XXX"){
```

Comment:
If you want to shorten this code, you can store the playerTurn variable in a winningCombo array like so:

winningCombo = playerTurn + playerTurn + playerTurn;

This will allow to check for winning string combinations without having to declare "OOO" and "XXX" literally.

Example:

//on lines 190-191

if(checkOne === winningCombo || checkTwo === winningCombo) {
    return true;
}